### PR TITLE
Feature/multi language support

### PR DESCRIPTION
**Pull request by MaximKulikov**
Add localization support for: default, se, ru languages with few examples.

Idea also fixed imports order to pass codacy code style:
groups imports: ensures that groups of imports come in a specific order (e.g., java. comes first, javax. comes second, then everything else)

http://checkstyle.sourceforge.net/config_imports.html#ImportOrder

### CODE REVIEW

File: res/bundle/strings_se.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿# Svensk lokalisering. Do not remove first comment line, coz app dont see first line key for no reason and crash....menu.bar.newPlaylist=Ny spellista
```
Comment (brokenprogrammer) [inline res/bundle/strings_se.properties]: Swedish localization is sv not se.
I tested your PR and it first didn't work but after i wrote getResourseBundle("sv") I finally got the Swedish translation.

**Review by brokenprogrammer — CHANGES_REQUESTED** [review]
I found one thing that needed changing otherwise your code was all good!
Thank you a lot for your contribution Trinion!

File: res/bundle/strings_se.properties (hunk 1 above)
Comment (MaximKulikov) [reply res/bundle/strings_se.properties]: Ops. Want me to fix it or you implement code youself?

Comment (brokenprogrammer) [reply res/bundle/strings_se.properties]: You can fix it, It's a small change and I have some other things that needs to be done :)
Just push your new changes into this branch and this PR will automatically be updated.

Comment (MaximKulikov) [reply res/bundle/strings_se.properties]: okie

Comment (MaximKulikov) [reply res/bundle/strings_se.properties]: You can override Sysytem user.language by run application with vm option "-Duser.language=sv"
new commit on the way

Comment (brokenprogrammer) [reply res/bundle/strings_se.properties]: You mean for testing? Like if I want to test the russian translation I would do "-Duser.language=ru" right? For me my System has the default language set to sv.

**Review by brokenprogrammer — APPROVED** [review]
Changes looks really good!
Thank you for your contribution Trinion!

**Pull request merged**